cmds.go

```diff
@@ -17,3 +17,9 @@ import "github.com/rueian/rueidis/internal/cmds"
 //   }
 // However, please know that once commands are processed by the Do() or DoMulti(), they are recycled and should not be reused.
 type Commands []cmds.Completed
+
+type (
+	Arbitrary = cmds.Arbitrary
+	Completed = cmds.Completed
+	Cacheable = cmds.Cacheable
+)
```

Comment:
Could you show me how do you want to access them? These lines look redundant to me.